static/css/layout/index.less

```diff
@@ -11,6 +11,11 @@ div#contentBody,
 div.contentBody {
   padding: 0 20px 20px;
 }
+div#contentBody.type--page {
+  img {
+    max-width: 100%;
```

Comment:
Maybe `max-width: 100%` makes sense as a general rule on `div.contentBody`? And thus wouldn't need a special class?

Reply:
I think indeed this is sufficient (and fixes other similar issues w/ image size, too):

```
diff --git a/static/css/components/page-history.less b/static/css/components/page-history.less
index b93db2c..24b1030 100644
--- a/static/css/components/page-history.less
+++ b/static/css/components/page-history.less
@@ -6,7 +6,6 @@

 // stylelint-disable-next-line selector-max-specificity
 #pageHistory {
-  width: 100%;
   margin-top: 30px;
 }
 .pageHistory__list {
diff --git a/static/css/layout/index.less b/static/css/layout/index.less
index 732ce53..197fb8f 100644
--- a/static/css/layout/index.less
+++ b/static/css/layout/index.less
@@ -10,6 +10,9 @@
 div#contentBody,
 div.contentBody {
   padding: 0 20px 20px;
+  img {
+    max-width: 100%;
+  }
 }
 .section,
 section {
```